testsuite/features/build_validation/init_clients/proxy_container.feature

```diff
@@ -33,12 +35,10 @@ Feature: Setup containerized proxy
 
 # workaround for bsc#1218146
 # Once we start using Leap Micro #23811 in Uyuni Proxy, we need to remove this Cucumber tag
-@skip
```

Comment:
Keep Proxy Host onboarding skipped for now please. Otherwise it will fail.

Reply:
Fixed.